crates/rocksdb-store/src/lib.rs

```diff
@@ -99,3 +113,72 @@ impl DbOpsConfig {
         Self { retry_count }
     }
 }
+
+// Opens rocksdb database instance from datadir
+pub fn open_rocksdb_database(
+    datadir: &Path,
+    dbname: &'static str,
+) -> anyhow::Result<Arc<rockbound::OptimisticTransactionDB>> {
+    let mut database_dir = datadir.to_path_buf();
+    database_dir.push("rocksdb");
+
+    if !database_dir.exists() {
+        fs::create_dir_all(&database_dir)?;
+    }
+
+    let mut opts = rockbound::rocksdb::Options::default();
+    opts.create_if_missing(true);
+    opts.create_missing_column_families(true);
+
+    let rbdb = rockbound::OptimisticTransactionDB::open(
+        &database_dir,
+        dbname,
+        STORE_COLUMN_FAMILIES.iter().map(|s| s.to_string()),
+        &opts,
+    )
+    .context("opening database")?;
+
+    Ok(Arc::new(rbdb))
+}
+
+pub type CommonDb =
+    CommonDatabase<L1Db, L2Db, SyncEventDb, ClientStateDb, ChainstateDb, RBCheckpointDB>;
```

Comment:
I would like to get rid of this `CommonDatabase` abstraction since it's not useful anymore.  But since there's a lot of stuff still using the `Database` trait I'm not sure how feasible that is yet.

Reply:
The feasibility is yet to be assessed. Once things are in place where they should be,  we can then get rid of the stuffs.